packages/kolibri-common/components/SyncSchedule/ManageSyncSchedule.vue

```diff
@@ -0,0 +1,365 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="goBack"
+  >
+    <KPageContainer>
+      <KGrid gutter="48">
+
+        <KGridItem>
+          <h1>{{ $tr('manageSyncTitle') }}</h1>
+
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ $tr('introduction') }}</p>
+        </KGridItem>
+
+
+        <KGridItem :layout8="{ span: 4 }" :layout12="{ span: 6 }" class="separate">
+          <b>{{ facility.name }}</b>
```

Comment:
`facility` starts out `null`, so when we initially try to access `facility.name` here, it gives an error.

We could perhaps conditionalize the `b` on `facility`:
```
<b v-if="facility">{{ facility.name }}</b>
<KCircularLoader v-else />
```
and add an else for a KCircularLoader to indicate to the user that loading is happening in the meantime.